test-data/run.test

```diff
@@ -506,37 +524,79 @@ def exc_info() -> Any:
 
 [file driver.py]
 import traceback
-from native import g, f, h, i
+import sys
+from native import g, f, h, i, j, k
+print("== i ==")
+try:
+    i()
+except:
+    traceback.print_exc(file=sys.stdout)
+
+print("== k ==")
+try:
+    k()
+except:
+    traceback.print_exc(file=sys.stdout)
+
+print("== g ==")
 g(True)
 g(False)
 
+print("== f ==")
 f(True)
 f(False)
 
+print("== h ==")
 h()
 
+print("== j ==")
+j(0)
+j(2)
+j(3)
 try:
-    i()
+    j(1)
 except:
-    traceback.print_exc()
+    print("out!")
 [out]
+== i ==
+<class 'IndexError'>
 Traceback (most recent call last):
-  File "tmp/driver.py", line 12, in <module>
+  File "tmp/driver.py", line 6, in <module>
     i()
-  File "tmp/py/native.py", line 37, in i
-    r(True)
+  File "tmp/py/native.py", line 42, in i
+    r(0)
   File "tmp/py/native.py", line 15, in r
-    x[1]
+    [0][1]
 IndexError: list index out of range
+== k ==
+Exception: hi
```

Comment:
Should we display a traceback for this exception as well? That's how normal Python exceptions seem to be reported.

Reply:
Fixed